src/zh/yidan/src/eu/kanade/tachiyomi/extension/zh/yidan/Yidan.kt

```diff
@@ -132,6 +132,6 @@ class Yidan : HttpSource(), ConfigurableSource {
 
     companion object {
         private const val MIRROR_PREF = "MIRROR"
-        private val MIRRORS get() = arrayOf("ydan.cc", "ydan.vip", "dans.cc")
+        private val MIRRORS get() = arrayOf("ydan.cc", "ydan.vip", "dans.cc", "yidan.one", "yidan.in", "yidan.info")
```

Comment:
```suggestion
        private val MIRRORS get() = arrayOf("ydan.cc", "yidan.one", "yidan.in", "yidan.info")
```

Remove dead domains.